### ISSUE TITLE
Add simple description of what this plugin does to readme

### ISSUE THREAD

**Issue by pReya** [lukaskleinschmidt/kirby-laravel-vite#3]
I came across this plugin, read through the `readme.md` and I still have no idea what it does (even though I'm familiar with Kirby, Laravel and Vite). It would be really nice, if the `readme.md` and the Github repository description contained a very brief 1-2 sentence summary of "why/what this plugin does".

**Comment by lukaskleinschmidt** [comment]
Fair point. I'll look into it after my vacation. Probably something along the lines of [this](https://laravel.com/docs/11.x/vite#introduction) tailored to the actual context would work?

**Comment by pReya** [comment]
@lukaskleinschmidt In the same vein as the original post here. I just tried to install the plugin (which works fine), but after installation there's not much documentation on how to setup/start a custom frontend/vite project.

For me, it's not clear, where to run the `npm install` command and where to put the given vite config file – since I don't have an existing frontend/vite project, yet (starting with a fresh Kirby plainkit install).

Would really appreciate some mor documentation there. Would love to help, once I figured out how to actually use it ;)

**Comment by lukaskleinschmidt** [comment]
Sorry I totally forgot about this issue.
I quickly put together a adapted [startkit](https://github.com/getkirby/starterkit) using vite for bundling:
https://github.com/lukaskleinschmidt/vite-starterkit

Have a look at my [commit](https://github.com/lukaskleinschmidt/vite-starterkit/commit/d6c9239) and the files I changed to make it work. To use the startkit follow these steps:

### 1. Install the Plugin
I added the plugin to the composer file so you can simply run the composer command to install it.
```
composer update
```
You could also just download the `kirby-laravel-vite` plugin into `/site/plugins/laravel-vite`.

### 2. Install NPM Dependencies
```
npm install
```

### 3. Using Vite
For local development you can use the following command. This will spawn a local dev server that will serve your assets including HMR or auto refreshing.
```
npm run dev
```

For production use this command to build the assets.
```
npm run build
```

----

Hope this will get you started. I'm all for improving the docs and would welcome any suggestions.
There are different ways to get `vitejs` running. You could for example also `pnpm`,  `deno` or `bun` (I probably missed a few).
But having an easy entry like the startkit might be a good solution?
